packages/cli/src/base.ts

```diff
@@ -41,6 +43,13 @@ export abstract class BaseCommand extends Command {
     return this._kit
   }
 
+  get metadataManager() {
```

Comment:
i think this should not be accessible from kit, instead have a module elsewhere that you can import from contractKit to use it. Since it's probably not interesting for most of our users.

Same as we did with BlockExplorer